lib/home/profile/profile.dart

```diff
@@ -99,36 +81,78 @@ class ProfileScreen extends StatelessWidget {
-          Opacity(opacity: 0.5, child: Text('email.address@gmail.com')),
-          SizedBox(height: 25),
-        ],
+    final userWatch = context.watch<AccountFields>().state!;
+    final linkColor = Color.lerp(ThcColors.dullBlue, ThcColors.teal, 0.25)!;
+    final overview = DefaultTextStyle(
+      style: TextStyle(height: 1.75, color: context.colorScheme.onBackground),
+      child: Center(
+        child: Column(
+          children: [
+            image,
+            Text(userWatch.name, style: const TextStyle(fontSize: 28)),
+            if (user!.id case final id?)
+              Text('user ID: $id', style: const TextStyle(fontWeight: FontWeight.w600)),
+            if (userWatch.email case final email?)
+              Text(email, style: TextStyle(color: linkColor)),
+            if (userWatch.phone case final phone?) Opacity(opacity: 0.75, child: Text(phone)),
+            const SizedBox(height: 25),
+          ],
+        ),
       ),
     );
 
-    return Scaffold(
-      body: Padding(
+    return ProfileListView(
+      itemCount: ProfileOption.values.length + 1,
+      itemBuilder: (_, index) => switch (index - 1) {
+        -1 => overview,
+        final i => ProfileOption.values[i],
+      },
+    );
+  }
+}
+
+class ProfileListView extends StatelessWidget {
+  const ProfileListView({required this.itemCount, required this.itemBuilder, super.key});
+  final int itemCount;
+  final NullableIndexedWidgetBuilder itemBuilder;
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: double.infinity,
+      child: SingleChildScrollView(
         padding: const EdgeInsets.all(15),
         child: Center(
           child: ConstrainedBox(
             constraints: const BoxConstraints(maxWidth: 600),
             child: ListView.separated(
-              itemCount: ProfileOption.values.length + 1,
-              separatorBuilder: (_, index) => const Divider(),
-              itemBuilder: (_, index) => switch (index - 1) {
-                -1 => overview,
-                final i => ProfileOption.values[i],
-              },
+              shrinkWrap: true,
+              physics: const NeverScrollableScrollPhysics(),
+              itemCount: itemCount,
+              itemBuilder: itemBuilder,
+              separatorBuilder: (_, __) => const Divider(),
             ),
           ),
         ),
       ),
     );
   }
 }
+
+class AccountFields extends Cubit<ThcUser?> {
+  AccountFields() : super(user);
+
+  void update(AccountField field) => emit(AccountField.updatedUser);
+
+  bool get hasChanges => AccountField.values.any((value) => value.updated != null);
+
+  Future<void> save(ThcUser updatedUser) async {
+    await updatedUser.upload();
+    emit(user = updatedUser);
+  }
+
+  Future<void> yeet(AccountField field) async {
+    final data = user!.json..remove(field.name);
+    await save(ThcUser.fromJson(data));
+  }
+}
```

Comment:
I have some mixed feelings about this class, since it's basically just a wrapper around the global `user` variable, and the implementation needs a `PopScope` in **account_settings.dart**.

I may try refactoring at some point in the future.